diff --git a/module-webif.c b/module-webif.c
index c4dd10ff..66178845 100644
--- a/module-webif.c
+++ b/module-webif.c
@@ -4004,7 +4004,7 @@ static char *send_ncam_user_config(struct templatevars *vars, struct uriparams *
 			}
 
 			lastresponsetm = latestclient->cwlastresptime;
-			if(latestclient->ip) {
+			if(latestclient!= NULL) {
 				connected_users += 1;
 				tpl_addVar(vars, TPLADD, "CLIENTIP", cs_inet_ntoa(latestclient->ip));
 				if(isactive > 0 || conn > 0)
