diff -Naur enigma2-PLi/lib/dvb/amldecoder.cpp enigma2-PersianPrince/lib/dvb/amldecoder.cpp
--- enigma2-PLi/lib/dvb/amldecoder.cpp	Thu Jan 24 14:00:39 2019
+++ enigma2-PersianPrince/lib/dvb/amldecoder.cpp	Fri Jan 25 18:34:05 2019
@@ -222,8 +222,13 @@
 				m_codec.video_type = VFORMAT_MPEG4; //maybe?
 				t = "MPEG4 Part2";
 				break;
+			case H265_HEVC:
+				m_codec.video_type = VFORMAT_HEVC;
+				t = "H265 HEVC";
+				break;
 		}
 		eDebug("[eAMLTSMPEGDecoder::setVideoPID] vpid=%d, type=%d %s", vpid, type, t.c_str());
+		aml_change_vpid(vpid, m_codec.video_type);
 	}
 	return 0;
 }
@@ -249,6 +254,10 @@
 				m_codec.audio_type = AFORMAT_AC3;
 				t = "AC3";
 				break;
+			case aDDP:
+				m_codec.audio_type = AFORMAT_AC3;
+				t = "AC3";
+				break;
 			case aAAC:
 				m_codec.audio_type = AFORMAT_AAC;
 				t = "AAC";
@@ -752,4 +761,11 @@
 		return -1;
 
 	return m_aspect == 1 ? 2 : 3;
+}
+
+int eAMLTSMPEGDecoder::getVideoGamma()
+{
+	//if (m_video)
+	//	return m_video->getGamma();
+	return -1;
 }
diff -Naur enigma2-PLi/lib/dvb/amldecoder.h enigma2-PersianPrince/lib/dvb/amldecoder.h
--- enigma2-PLi/lib/dvb/amldecoder.h	Thu Jan 24 14:00:39 2019
+++ enigma2-PersianPrince/lib/dvb/amldecoder.h	Fri Jan 25 18:34:05 2019
@@ -89,7 +89,7 @@
 	void *adec_handle;
 
 public:
-	enum { aMPEG, aAC3, aDTS, aAAC, aAACHE, aLPCM, aDTSHD, aDDP, MPEG2 = 0, MPEG4_H264, MPEG1, MPEG4_Part2, VC1, VC1_SM };
+	enum { aMPEG, aAC3, aDTS, aAAC, aAACHE, aLPCM, aDTSHD, aDDP,UNKNOWN = -1, MPEG2=0, MPEG4_H264, VC1 = 3, MPEG4_Part2, VC1_SM, MPEG1, H265_HEVC, AVS = 16 };
 	enum { pidNone = -1 };
 	eAMLTSMPEGDecoder(eDVBDemux *demux, int decoder);
 	virtual ~eAMLTSMPEGDecoder();
@@ -141,6 +141,7 @@
 	int getVideoProgressive();
 	int getVideoFrameRate();
 	int getVideoAspect();
+	int getVideoGamma();
 	static RESULT setHwPCMDelay(int delay);
 	static RESULT setHwAC3Delay(int delay);
 };
