diff -Naur servicemp3-PLi/configure.ac servicemp3-PersianPrince/configure.ac
--- servicemp3-PLi/configure.ac	2019-01-20 17:06:12.000000000 +0330
+++ servicemp3-PersianPrince/configure.ac	2019-02-11 14:07:27.001713362 +0330
@@ -16,6 +16,25 @@
 
 PKG_CHECK_MODULES(GSTREAMER, gstreamer-$GST_MAJORMINOR gstreamer-pbutils-$GST_MAJORMINOR)
 
+AC_ARG_WITH(amlogic,
+	AC_HELP_STRING([--with-amlogic], [use amlogic MPEG decoder]),
+	[[with_amlogic=$withval]],
+	[[with_amlogic=no]]
+)
+AM_CONDITIONAL(HAVE_AMLOGIC, test x"$with_amlogic" = xyes)
+if test x"$with_amlogic" = xyes ; then
+	AC_DEFINE(HAVE_AMLOGIC, 1,[Enables AMLOGIC decoder support])
+fi
+
+if test "$BOXTYPE" == "alien5"; then
+	AC_DEFINE(HAVE_ALIEN5, 1,[enable Alien5 specific hacks])
+fi
+
+if test "$BOXTYPE" == "k1plus" -o "$BOXTYPE" == "k1pro" -o "$BOXTYPE" == "k2pro" -o "$BOXTYPE" == "k2prov2" -o "$BOXTYPE" == "k3pro" -o "$BOXTYPE" == "kvim2"; then
+	AC_DEFINE(HAVE_ALIEN5, 1,[enable Alien5 specific hacks])
+	AC_DEFINE(HAVE_MECOOL, 1,[enable Mecool devices specific hacks])
+fi
+
 AC_DEFINE([DEBUG])
 
 AC_CONFIG_FILES([
diff -Naur servicemp3-PLi/servicemp3/servicemp3.cpp servicemp3-PersianPrince/servicemp3/servicemp3.cpp
--- servicemp3-PLi/servicemp3/servicemp3.cpp	2019-01-20 17:06:12.000000000 +0330
+++ servicemp3-PersianPrince/servicemp3/servicemp3.cpp	2019-02-11 14:05:42.952461933 +0330
@@ -21,6 +21,12 @@
 #include <gst/pbutils/missing-plugins.h>
 #include <sys/stat.h>
 
+#if HAVE_ALIEN5
+extern "C" {
+#include <codec.h>
+}
+#endif
+
 #define HTTP_TIMEOUT 30
 
 /*
@@ -624,6 +630,17 @@
 #endif
 	if ( m_gst_playbin )
 	{
+#if HAVE_ALIEN5
+		aml_set_mediaplay_source((void *)m_gst_playbin,(int)m_sourceinfo.is_audio);
+#endif
+#if HAVE_MECOOL
+		/*
+		* test forcing aml-video-sink
+		*/
+		videoSink = gst_element_factory_make("amlvsink", "GstDVBVideoSink");
+		g_object_set (m_gst_playbin, "video-sink", videoSink, NULL);
+		eDebug("setting video-sink=amlvsink");
+#endif
 		/*
 		 * avoid video conversion, let the dvbmediasink handle that using native video flag
 		 * volume control is done by hardware, do not use soft volume flag
